src/components/forms/DateInputGroup/DateInputGroup.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { DateInputGroup } from './DateInputGroup'
+
+export default {
+  title: 'Form/DateInputGroup',
+  parameters: {
+    info: `
+USWDS 2.0 DateInput component
+
+Source: https://designsystem.digital.gov/components/form-controls/#date-input
+`,
+  },
+}
+
+// TODO Determine if this is the way to display this component
+export const defaultDateInputGroup = (): React.ReactElement => (
```

Comment:
Since `DateInputGroup` on its own doesn't do much I wouldn't say we _need_ to have stories for it